src/panels/lovelace/cards/hui-picture-glance-card.js

```diff
@@ -69,7 +70,7 @@ class HuiPictureGlanceCard extends NavigateMixin(LocalizeMixin(EventsMixin(Polym
 
       <ha-card>
         <hui-image
-          class$='[[_computeImageClass(_config)]]'
+          class$='[[_computeImageClass]]'
```

Comment:
This function will not be called. And if it _would_ be called, it would not be called again when config changes.